react-native/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -33,11 +33,6 @@
     @TargetApi(Build.VERSION_CODES.KITKAT)
     protected void onCreate(Bundle savedInstanceState) {
         logFile = this.getFileStreamPath("android.log");
-        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M && !Settings.canDrawOverlays(this) && this.getUseDeveloperSupport()) {
```

Comment:
seemingly works fine without this. This is setup in other RN code so i think we don't need to do this ourselves anymore